asm/dolphin/os/OSResetSW_data.s

```diff
@@ -1,19 +0,0 @@
-.include "macros.inc"
```

Comment:
GitHub doesn't say this was deleted, is it an empty file now?

Reply:
Huh, weird